src/v/raft/tests/raft_fixture.h

```diff
@@ -50,6 +50,7 @@
 
 #include <optional>
 #include <ranges>
+#include <system_error>
```

Comment:
whats this selective cherry-pick?

Reply:
The original commit https://github.com/redpanda-data/redpanda/commit/d1ecf0e4debae17fb500a8708f38ae5c0314baa5 has this addition (not sure whether for a good reason or not). It has been already backported to v.24.2.x, but without this line. Now cherry-picking it again resulted in this line only.

I personally think that backports should be as close to the original as possible to preserve similarity between branches. If we decide to get rid of this include we should start with dev branch. So I don't mind this change.